next-cloudinary/src/types/player.ts

```diff
@@ -3,20 +3,71 @@ export interface CloudinaryVideoPlayer {
 }
 
 export interface CloudinaryVideoPlayerOptions {
-  autoplayMode?: string;
-  cloud_name?: string;
+  // ------------ Player visuals Props ------------
+  aiHighlightsGraph?: boolean;
+  bigPlayButton?: boolean | string;
   colors?: CloudinaryVideoPlayerOptionsColors;
+  controlBar?: {
+    pictureInPictureToggle?: boolean;
+    chaptersButton?: boolean;
+  };
   controls?: boolean;
+  floatingWhenNotVisible?: string;
+  fluid?: boolean;
   fontFace?: string;
+  hideContextMenu?: boolean;
+  interactionAreas?: any; // Used "any" cause this fetaure is still in beta
+  playbackRates?: Array<any>;
+  playlistWidget?: {
+    direction?: string;
+    total?: string
+  };
+  posterOptions?: object;
+  showJumpControls?: boolean;
+  // showLogo?: boolean; // Comment This Out cause it seems to be a duplicate of "CldVideoPlayerPropsLogo"
```

Comment:
so i think we need these still. this specific file player.ts represents the types of the options passed into the `videoPlayer` method
<img width="906" alt="image" src="https://github.com/colbyfayock/next-cloudinary/assets/1045274/e59cc0bf-00fc-48e9-a783-6b31bf5dd858">

i see what you mean though about it existing in `CloudinaryVideoPlayerOptionsLogo` but that isn't referenced on this `CloudinaryVideoPlayerOptions` type at all

perhaps you're more experienced with TS than i am, does that mean we need to add this?

```
export interface CloudinaryVideoPlayerOptions & CldVideoPlayerPropsLogo {
```

i had them separately defined because im using it when defining the logoOptions inside the player component

Reply:
Just to be sure in your code example snippet did you mean to use ````CloudinaryVideoPlayerOptionsLogo```` instead of ````CldVideoPlayerPropsLogo```` ? 

If you were then yes, you are actually on the right path. But we need to actually add this since its an interface:
```` 
export interface CloudinaryVideoPlayerOptions extends CloudinaryVideoPlayerOptionsLogo{
````

Reply:
oops you're right, `CloudinaryVideoPlayerOptions` and `CloudinaryVideoPlayerOptionsLogo`

that makes sense then. im not sure what best practice is for breaking out types into their own objects like that, but probably rethinking that is out of scope, for now at least making sure those types are correctly defined on the `CloudinaryVideoPlayerOptions` will work

Reply:
Well in my opinion you are actually doing things the right way